rakelib/example_type.rb

```diff
@@ -0,0 +1,160 @@
+require "rake"
```

Comment:
@justin808 because it defines a class called `ExampleType`, which are objects that hold data about the types of examples we want to generate